### PR TITLE
Add README file

### PR DESCRIPTION
**Pull request by tonyredondo**

### CODE REVIEW

**Review by AdrianLC — APPROVED** [review]
Looks very clear to me

File: README.md [hunk 1]
```diff
@@ -1,5 +1,161 @@
...
+```shell
+go get -u github.com/DataDog/dd-sdk-go-testing
+```
+
+#### Requires:
+- Go >= 1.12
+- Datadog's Trace Agent >= 5.21.1
+
+### Instrumenting your tests
+In order to instrument your tests that use Go's native
```
Comment (juan-fernandez) [inline README.md:18]: "your tests that use" sounds a bit weird.
I'd probably go for 
```suggestion
To instrument tests that use Go's native 
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,161 @@
...
+
+For example:
+
+```go
+package go_sdk_sample
+
+import (
+	"os"
+	"testing"
+
```
Comment (juan-fernandez) [inline README.md:32]: is this empty line intended?

File: README.md [hunk 3]
```diff
@@ -1,5 +1,161 @@
...
+			defer finish()
+
+			// Test code ...
+		})
+	}
+}
+```
+
+Note that after this, you can use `ctx` to refer to the context of the running test, which has information
+about its trace. Use it when you make any external calls in order to see the traces within the test span.
```
Comment (juan-fernandez) [inline README.md:75]:
```suggestion
about its trace. Use it when you make any external call to see the traces within the test span.
```

File: README.md [hunk 4]
```diff
@@ -1,5 +1,161 @@
...
+```go
+package go_sdk_sample
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
```
Comment (juan-fernandez) [inline README.md:88]: I guess this is for making the relevant lines more prominent?

**Review by juan-fernandez — APPROVED** [review]
nice! Only nitpicks

File: README.md (hunk 2 above)
Comment (tonyredondo) [reply README.md:32]: yes, to make the separation between internal packages and external modules.

File: README.md (hunk 4 above)
Comment (tonyredondo) [reply README.md:88]: in go you normally want to group imports.

Comment (juan-fernandez) [reply README.md:88]: cool, thanks!

File: README.md [hunk 5]
```diff
@@ -1,5 +1,161 @@
...
-Datadog Test Instrumentation framework for Go
+> Datadog Test Instrumentation framework for Go
 
-# Status
-Work in progress
\ No newline at end of file
+## Getting Started
+
+### Installing
+Installation of the DataDog go testing sdk is done via `go get`:
```
Comment (fermayo) [inline README.md:7]:
```suggestion
Installation of the Datadog Go testing SDK is done via `go get`:
```

**Review by fermayo — APPROVED** [review]

**Pull request merged**